src/main/java/Duke.java

```diff
@@ -1,10 +1,125 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public static void printGreeting() {
+        String logo = "    *******     *******   *******       **\n"
+                + "    /**////**   **/////** /**////**     ****\n"
+                + "    /**    /** **     //**/**   /**    **//**\n"
+                + "    /**    /**/**      /**/*******    **  //**\n"
+                + "    /**    /**/**      /**/**///**   **********\n"
+                + "    /**    ** //**     ** /**  //** /**//////**\n"
+                + "    /*******   //*******  /**   //**/**     /**\n"
+                + "    ///////     ///////   //     // //      //\n";
+
+        System.out.println("    ____________________________________________________________");
+        System.out.println(logo + "\n    Kon'nichiwa! Doraemon desu.\n    What can I do for you?");
+        System.out.println("    ____________________________________________________________");
+    }
+
+    public static Command getCommand(String line) {
+        Command command;
+        if (line.equals("bye")) {
+            command = Command.BYE;
+        } else if (line.equals("list")) {
+            command = Command.LIST;
+        } else if (line.startsWith("mark ")) {
+            command = Command.MARK;
+        } else if (line.startsWith("unmark ")) {
+            command = Command.UNMARK;
+        } else if (line.startsWith("todo ")) {
+            command = Command.TODO;
+        } else if (line.startsWith("deadline ")) {
+            command = Command.DEADLINE;
+        } else if (line.startsWith("event ")) {
+            command = Command.EVENT;
+        } else {
+            command = Command.EMPTY;
+        }
+        return command;
+    }
```

Comment:
interesting way to read the input - i simply used a split function to do it

Reply:
Thanks for suggesting the split function! Have decided to use it in my latest increment.